### PR TITLE
Add get_patient()

### PR DESCRIPTION
**Pull request by dkeden**
Add function to query patient w/ a given `patient_id`. This function and query is similar to `get_patients()`.

Useful for retrieving the patient's user ID, name and email address.

Being that this is returning user information however, it may be more informative to rename to `get_user()` or `get_user_from_patient_id()`. Thoughts?

### CODE REVIEW

File: seerpy/seerpy.py [hunk 1]
```diff
@@ -500,6 +500,22 @@ def get_organisations_dataframe(self):
             return orgs
         return pd.DataFrame(orgs)
 
+    def get_patient(self, patient_id):
+        """
+        Get user ID and info from patient ID.
+        Parameters
+        ----------
+        patient_id : str
+            The Seer patient ID for which to retrieve diary labels
```
Comment (bjdoyle) [inline seerpy/seerpy.py:509]: remove Seer, it's implied and a bit confusing.

Comment (dkeden) [reply seerpy/seerpy.py:509]: Okay so `get_patient()` is too ambiguous? I'll go with `get_user_for_patient()`. Good idea for the DataFrame, I'll do that now

Comment (dkeden) [reply seerpy/seerpy.py:509]: Do you think `get_user_from_patient_id()` would be OK? I think I would remember it better, but that's not a terribly good reason to up vote a suggestion 😄

Comment (bjdoyle) [reply seerpy/seerpy.py]: I like get_user_for_patient better as it describes what is happening, and the thing you use to get it is the id, but you could potentially do that by some other attribute later if that made sense.

Comment (dkeden) [reply seerpy/seerpy.py]: Okay, fair enough!

Comment (dkeden) [reply seerpy/seerpy.py]: Done!

**Comment by dkeden** [comment]
Good to learn some German on the way too. Sounds good. Let me know the way forward for that & if / when I can help

**Pull request merged**